go.mod

```diff
@@ -1,17 +1,19 @@
 module github.com/NVIDIA/k8s-operator-libs
 
-go 1.20
+go 1.21
+
+toolchain go1.21.5
 
 require (
-	github.com/go-logr/logr v1.2.4
-	github.com/onsi/ginkgo/v2 v2.11.0
-	github.com/onsi/gomega v1.27.10
+	github.com/go-logr/logr v1.4.1
+	github.com/onsi/ginkgo/v2 v2.14.0
+	github.com/onsi/gomega v1.30.0
 	github.com/stretchr/testify v1.8.4
-	k8s.io/api v0.28.3
-	k8s.io/apimachinery v0.28.3
-	k8s.io/client-go v0.28.3
-	k8s.io/kubectl v0.28.3
-	sigs.k8s.io/controller-runtime v0.16.3
+	k8s.io/api v0.29.0
+	k8s.io/apimachinery v0.29.0
+	k8s.io/client-go v0.29.0
+	k8s.io/kubectl v0.29.0
```

Comment:
Note: This is needed in order to bump network-operator to CR 0.17.0 as there are conflicting dependencies.